### PR TITLE
Moves previous code to _archive and updates README

### PR DESCRIPTION
**Pull request by Tmm0k**
All I did to the README was replace audit-cli with audit-bot. We can edit this more if you would like. 💯 @ekdevdes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# audit-cmd
+# audit-bot
                 
-[![Known Vulnerabilities](https://snyk.io/test/github/ekdevdes/audit-cli/badge.svg?targetFile=package.json)](https://snyk.io/test/github/ekdevdes/audit-cli?targetFile=package.json)
+[![Known Vulnerabilities](https://snyk.io/test/github/ekdevdes/audit-bot/badge.svg?targetFile=package.json)](https://snyk.io/test/github/ekdevdes/audit-bot?targetFile=package.json)
```
Comment (ekdevdes) [inline README.md:3]: This will break the badge for now until we rename the repo

**Review by ekdevdes — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 A command line tool that allows to run Google Lighthouse and Mozilla Observatory tests for security, best practices, SEO, PWA compliance, and accesibility on local and public urls
 
 ## Installation
```
Comment (ekdevdes) [inline README.md]: @constsync Can you remove "Installation" and "Usage" sections

**Review by ekdevdes — CHANGES_REQUESTED** [review]
Can you remove the "Installation" and "Usage" sections for now?

File: README.md (hunk 1 above)
Comment (Tmm0k) [reply README.md:3]: Yeah, true, I think you're gonna have to do that though. I don't see the settings tab 😂 🙃 I guess we can merge this after we change the repo name. @ekdevdes

Comment (ekdevdes) [reply README.md:3]: Done.

**Review by ekdevdes — APPROVED** [review]

**Pull request merged**